d3a_api_client/setups/extract_area_uuid_from_area_name.py

```diff
@@ -9,7 +9,7 @@
 
 # set simulation parameters
 simulation_id = str(sys.argv[3])
-domain_name = 'https://d3aweb-dev.gridsingularity.com'
+domain_name = 'https://d3aweb.gridsingularity.com'
```

Comment:
Ideally this is an environmental variable so the dev tester can set it too, right?

Reply:
Or even a CLI parameter would be good (if this script is used often).

Reply:
Using default domain name from env, if user like can simply overwrite the env var: https://github.com/gridsingularity/d3a-api-client/pull/150/commits/5eaa2c5a78830404fbad1444436fe32c6fd36f0b